testing/internal/e2e/tests/base/target_tcp_connect_golang_ssh_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package base_test
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"golang.org/x/crypto/ssh"
+
+	"github.com/hashicorp/boundary/internal/target"
+	"github.com/hashicorp/boundary/testing/internal/e2e"
+	"github.com/hashicorp/boundary/testing/internal/e2e/boundary"
+)
+
+// TestCliTcpTargetConnectGolangSsh uses the golang ssh library to connect to a target
+func TestCliTcpTargetConnectGolangSsh(t *testing.T) {
```

Comment:
Pettiest of complaints but I dislike the use of `golang` generally as a phrase. These set of repos are generally known as "experimental" or "sub-repositories": https://pkg.go.dev/golang.org/x. The `golang` in the URL is out of necessity, the name is Go. What do you think about
```suggestion
// TestCliTcpTargetConnectGoSsh uses the go experimental ssh library to connect to a target
func TestCliTcpTargetConnectGoSsh(t *testing.T) {
```

Would obviously mean renaming the file too.